backends/saves/savefile.cpp

```diff
@@ -56,6 +56,18 @@ int64 OutSaveFile::pos() const {
 	return _wrapped->pos();
 }
 
+bool OutSaveFile::seek(int64 offset, int whence) {
+	Common::SeekableWriteStream *sws =
+		dynamic_cast<Common::SeekableWriteStream *>(_wrapped);
```

Comment:
I don't quite understand this change. All of our `createWriteStream()` calls (used by `DefaultSaveFileManager::openForSaving()` are returning `Common::WriteStream`. How this cast could ever be executed?

Reply:
As I understand it (being at work, and away from PC), the OpenForSaving by default uses the StdioStream, which was previously changed to derive from SeekableWriteStream back when SeekableWriteStream was first introduced. So for Windows (and presumably Mac and Linux and others that use that standard implementation) the _wrapped writestream can be cast to SeekableWriteStream. It's just any backend, if any, that uses an alternative saving logic that would fail the cast.